c2rust-analyze/src/rewrite/expr/convert.rs

```diff
@@ -155,6 +155,19 @@ impl<'tcx> intravisit::Visitor<'tcx> for ConvertVisitor<'tcx> {
                     Rewrite::Ref(Box::new(self.get_subexpr(ex, 0)), mutbl_from_bool(*mutbl))
                 }
 
+                mir_op::RewriteKind::CastRefToRaw { mutbl } => {
+                    let rw_pl = Rewrite::Deref(Box::new(hir_rw));
+                    Rewrite::AddrOf(Box::new(rw_pl), mutbl_from_bool(*mutbl))
+                }
```

Comment:
If it's already a ref, why do we need to deref and addr_of vs. just casting to the ptr type?

Reply:
Added a comment.